### DISCUSSION TITLE
Can not access webRTC shows error : Deadline exceeded while waiting for connection

### DISCUSSION THREAD

**Discussion by jatinjotwani** [bluenviron/mediamtx#2872] [Questions]
### Question

All the things done below are down on aws server(lightsail)
I have pulled the bluenviron/mediamtx:latest image and started a container using this command:
sudo docker run -it --name stream -p 48554:8554 -p 48889:8889 -p 48888:8888 bluenviron/mediamtx:latest
And now I am publishing to a RTSP stream to port 48554 , and the container listens it successfully but as soon as I start accessing the webRTC on
 http://my_ip_address:48889/live
a error shows up :
Deadline exceeded while waiting for connection

**Comment by aler9** [comment]
please read here:

https://github.com/bluenviron/mediamtx#connectivity-issues

[a comment by an automated account is not shown]
